runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -1013,6 +1013,16 @@ J9::CodeGenerator::lowerTreeIfNeeded(
       self()->lowerNonhelperCallIfNeeded(node, tt);
       }
 
+   if (node->getOpCode().isCall() &&
+       !node->getSymbol()->castToMethodSymbol()->isHelper() &&
+       (node->getSymbol()->castToMethodSymbol()->getRecognizedMethod() == TR::java_lang_invoke_MethodHandle_invokeBasic))
```

Comment:
I assume code later will clean up the call?

Reply:
I am not sure what you mean by this comment. Please explain what you were expecting the later code to do.

Reply:
@andrewcraik The call will remain in the tree, codegen will generate instructions for it. It's just that we need to store additional information to J9VMThread, hence this PR